src/json/jobs/UserImportLine.schema.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "object",
```

Comment:
@jericmason this could use a `title` and `description` beyond just the name. I assume that I know what this is for, but that extra bit of documentation would be useful.

Reply:
e.g. this is for the column names in the CSV right?

Reply:
Title and description should be in there now.